src/identity-utils.ts

```diff
@@ -279,3 +280,28 @@ const getExpireTimestamp = (maxAge: number = ONE_DAY_IN_SECONDS): number =>
 
 const parseIdentityResponse = (responseText: string): IdentityResultBody =>
     responseText ? JSON.parse(responseText) : ({} as IdentityResultBody);
+
+export const hasIdentityRequestChanged = (
+    mpInstance: IMParticleWebSDKInstance
+): boolean => {
+    const currentUser = mpInstance.Identity.getCurrentUser();
+
+    const newIdentityRequest = mpInstance._Store.SDKConfig.identifyRequest;
```

Comment:
Let's avoid bringing in `mpInstance` if we can.

```suggestion
export const hasIdentityRequestChanged = (
    currentUser: MPUser, newIdentityRequest: IdentityRequest
): boolean => {
```

Then we can just set these outside the call:
```
    hasIdentityRequestChanged(
        mpInstance.Identity.getCurrentUser,
        mpInstance._Store.SDKConfig.identifyRequest;
    );
```

The benefit is that we now don't need to inject the whole SDK and testing should be easier.